app/models/katello/package_group.rb

```diff
@@ -1,7 +1,30 @@
 module Katello
-  class PackageGroup
+  class PackageGroup < Katello::Model
+    include Glue::Pulp::PulpContentUnit
     include Glue::Pulp::PackageGroup if Katello.config.use_pulp
-    include Glue::ElasticSearch::PackageGroup if Katello.config.use_elasticsearch
+
     CONTENT_TYPE = "package_group"
+
+    has_many :repositories, :through => :repository_package_groups, :class_name => "Katello::Repository"
```

Comment:
Can we add some validation that ensures that even though we are defining this as a `has_many` to avoid modifying other code (although, I'd really think the data layer should be the more correct setup instead of it being the work around) that we only ever assign a given package group to a single repository?